### PR TITLE
Pull jaeger, seq and splunk into separate compose files

### PR DESCRIPTION
**Pull request by pbolduc**
# Description

This PR includes the following proposed change(s):

- pulls jaeger, seq and splunk to separate compose files

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring / Documentation
- [ ] Version change

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,38 +50,14 @@ Welcome to Traffic Courts Online
...
 # Splunk
 
-The default `docker-compose.yaml` file contains a local splunk service. A custom configuration file, `./.docker/splunk-dev-config.yaml`
+The default `docker-compose.yaml` file does NOT contain Splunk configuration. To use/test with Splunk,
+you can include 
+
+A custom configuration file, `./.docker/splunk-dev-config.yaml`
 is used to adjust the default settings. A key setting is disabling the SSL on the HEC endpoint.
 
 https://splunk.github.io/docker-splunk/EXAMPLES.html#create-standalone-with-hec
```
Comment (KDNeufeld) [inline README.md:60]: I would be nice to include in the README example uses of jaeger, seq, etc.
Maybe just 
```
docker-compose -f docker-compose.yml -f ./docker/docker-compose.seq.yml up -d
```

**Review by KDNeufeld — APPROVED** [review]
Very nice.  Really simplifies docker-compose

File: README.md (hunk 1 above)
Comment (pbolduc) [reply README.md]: I actually had, but forgot to save/commit it. Pushed changes in last commit.

Comment (KDNeufeld) [reply README.md]: This looks much better. Thanks.

**Pull request merged**